source/index-warning.txt

```diff
@@ -57,37 +57,32 @@ field is used in a query, but the field isn't indexed in the database:
       Filters.ne( "awards", "Comedy" )
    )
 
-The plugin shows this warning:
+The side panel shows the following warning under 
+:guilabel:`Performance Warnings`:
 
-.. code-block:: none
-   :copyable: false
-
-   This query will run without an index. If you plan on using this
-   query heavily in your application, you should create an index that
-   covers this query.
-   
-   Implement an Index
+.. image:: /images/index-warning.png
+   :alt: Index warning in the IntelliJ Plugin.
+   :width: 400px
 
 Create an Index
 ~~~~~~~~~~~~~~~
 
-To create an index for the query, click the :guilabel:`Implement an
-Index` link displayed with the warning in the plugin.
+To create an index for the query, click the :guilabel:`Create an
```

Comment:
(Info typing)
This is technically a task. Any chance you can wrangle this into a procedure format? Possibly on its own page?

Reply:
I’m inclined to keep it on this page since the index creation is a part of the plugin/warning. However, I will make this into a procedure format!